internal/connect/command_custom_plugin_version_create.go

```diff
@@ -33,15 +33,15 @@ func (c *customPluginCommand) newVersionCreateCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Create custom connector plugin version for plugin "ccp-123456".`,
-				Code: "confluent connect custom-plugin version create --plugin ccp-123456 --plugin-file datagen.zip --version-number 0.0.1 --beta=true --sensitive-properties=passwords,keys,tokens",
+				Code: "confluent connect custom-plugin version create --plugin ccp-123456 --plugin-file datagen.zip --version-number 0.0.1 --sensitive-properties passwords,keys,tokens",
 			},
 		),
 	}
 
 	cmd.Flags().String("plugin", "", "ID of custom connector plugin.")
 	cmd.Flags().String("plugin-file", "", "Custom plugin ZIP or JAR file.")
 	cmd.Flags().String("version-number", "", "Version number of custom plugin version.")
-	cmd.Flags().Bool("beta", false, `Mark the custom plugin version as beta. (default "false")`)
+	cmd.Flags().Bool("beta", false, `Mark the custom plugin version as beta. (default "false").`)
```

Comment:
Do we really need to specify the default here? You would think that it would be implied since it is a boolean flag, after all.

Reply:
As a new user reading the help documentation ⬇️ , I might be confused about the --beta flag. While most other flags clearly show their type as ‘string,’ the --beta flag doesn’t specify a type. However, the default value of false suggests that this flag is likely a boolean. This could serve as a clue for me to understand its purpose.
```
Flags:
      --plugin string                  REQUIRED: ID of custom connector plugin.
      --version string                 REQUIRED: ID of custom connector plugin version.
      --version-number string          Version number of custom plugin version.
      --beta                           Mark the custom plugin version as beta. (default "false").
      --release-notes string           Release notes for custom plugin version.
      --sensitive-properties strings   A comma-separated list of sensitive property names.
      --context string                 CLI context name.
```

Reply:
I think the examples are a great way to communicate the usage of these flags! By the way, all of our other boolean flags have this formatting throughout the CLI. I don't think we necessarily need to make this flag a special case.

Reply:
If you disagree, feel free to keep the `(default "false")` here, but let's remove the period at the end to match the output of our other commands

Reply:
Gotcha, thank you! I totally agree. Previous I thought if we didn't have the beta example (to pass the lint in order to merge pr) it would be confusing.